tests/api.js

```diff
@@ -70,6 +73,10 @@ describe('Initialization', function() {
   });
 
   it('should detect missing cloudant', function(done) {
+    if (process.env.NOCK_OFF) {
+      this.skip();
```

Comment:
The `after` stuff just runs the test database deletes. I've run the tests numerous times now and we're not building up a collection of old `nodejs_cloudant_test_*` databases so I think we're okay.